EXILED/Exiled.Loader/Loader.cs

```diff
@@ -476,6 +450,48 @@ private static bool CheckPluginRequiredExiledVersion(IPlugin<IConfig> plugin)
             return false;
         }
 
+        /// <summary>
+        /// Load every plugin inside the given directory, if null it's default EXILED one (global).
+        /// </summary>
+        /// <param name="dir">The sub-directory of the plugin - if null the default EXILED one will be used.</param>
+        private static void LoadPluginsFromDirectory(string dir = null)
+        {
+            string path = Paths.Plugins;
+            if (dir != null)
+                path = Path.Combine(path, dir);
+
+            if (!Directory.Exists(path))
+                Directory.CreateDirectory(path);
+
+            foreach (string assemblyPath in Directory.GetFiles(path, "*.dll"))
+            {
+                Assembly assembly = LoadAssembly(assemblyPath);
+
+                if (assembly is null)
+                    continue;
+
+                Locations[assembly] = assemblyPath;
+            }
+
+            foreach (Assembly assembly in Locations.Keys)
+            {
+                if (Locations[assembly].Contains("dependencies"))
+                    continue;
+
+                IPlugin<IConfig> plugin = CreatePlugin(assembly);
+
+                if (plugin is null)
+                    continue;
+
+                AssemblyInformationalVersionAttribute attribute = plugin.Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>();
+
+                Log.Info($"Loaded plugin {plugin.Name}@{(plugin.Version is not null ? $"{plugin.Version.Major}.{plugin.Version.Minor}.{plugin.Version.Build}" : attribute is not null ? attribute.InformationalVersion : string.Empty)}");
```

Comment:
```suggestion
                Log.Info($"Loaded plugin {plugin.Name}@{(plugin.Version != null ? $"{plugin.Version}" : attribute != null ? attribute.InformationalVersion : string.Empty)}");
```

I don't know why this wouldn't just use the version ToString provided by Dotnet.

Reply:
the funniest part is that that code wasn't added by me :D